isolate_example/README.md

```diff
@@ -9,6 +9,27 @@ A sample application that demonstrate best practices when using [`isolates`](htt
 * Demonstrate how to initialize and use an isolate.
 * Show the cost of moving data between isolates and provide alternatives.
 
+## The important bits
+
+### `page_one.dart`
+
+Compares running a large computation on the main isolate vs on a secondary isolate. 
+The `SmoothAnimationWidget` will lower frames when the performance drops.
+
+### `page_two.dart`
+
+An infinite process will be run on the secondary isolate.  The process will be an 
```

Comment:
Technically the process isn't infinite, right? Consider:

"Creates an isolate and uses it to compute the sum of 100M randomly generated numbers while reporting progress. Users can start, terminate, pause, and resume the isolate, as well as modify how the calculation is performed."